### PR TITLE
FIX: #2898 `source prompt` throws an error

### PR DESCRIPTION
**Pull request by greggirwin**

### CODE REVIEW

File: modules/view/view.red [hunk 1]
```diff
@@ -854,7 +854,8 @@ dump-face: function [
 ][
 	depth: ""
 	print [
-		depth "Style:" face/type "Offset:" face/offset "Size:" face/size
+		depth "Type:" face/type "Style:" if face/options [face/options/style]
```
Comment (geekyi) [inline modules/view/view.red]: Looks like something else got fixed too :p

**Comment by greggirwin** [comment]
Yeah, the way github combines commits into open PRs makes it a bit of a pain sometimes, when I'd like to keep things separate.

**Comment by geekyi** [comment]
@greggirwin you'd have to make a new branch (I'm guessing)

**Comment by greggirwin** [comment]
Yes, which I will do if I have to, but is more work. I do try to keep from Making Doc's like miserable with my PRs, and have gotten better at that. :^)

**Pull request merged**